ui/apps/platform/src/Containers/Login/TestLoginResultsPage.js

```diff
@@ -93,7 +92,7 @@ function TestLoginResultsPage({ authProviderTestResults }) {
     const { messageBody, messageType } = getMessage(authProviderTestResults);
 
     return (
-        <AppWrapper>
+        <>
```

Comment:
Because `AppWrapper` element had only one child, fragment is redundant but intended as a minimal safe change because this page is very unfamiliar to me.

Reply:
This page is only used in Access section to check rules, I seem to remember.

Reply:
**Test login** button on Auth provider form?

Reply:
Yest that's the one.